pkg/webhooks/webhookpatch.go

```diff
@@ -75,12 +76,17 @@ func NewWebhookCertPatcher(
 		func(options *metav1.ListOptions) {
 			options.LabelSelector = fmt.Sprintf("%s=%s", label.IoIstioRev.Name, revision)
 		})
+	retryBackoff := backoff.NewExponentialBackOff()
+	retryBackoff.InitialInterval = 2 * time.Second
+	retryBackoff.MaxInterval = 15 * time.Minute // Max Backoff interval.
+	// If certificate has not been patched with in an hour of retries, let us stop retrying.
+	retryBackoff.MaxElapsedTime = 1 * time.Hour
```

Comment:
This means we may never recover from a temporary outage or permission error, I am not sure if that is safe. 15 mins is already really long internal anyhow?